app/domain/authentication/authn_jwt/input_validation/validate_claim_name.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Validate the claim name value
+      ValidateClaimName ||= CommandClass.new(
```

Comment:
it's quite hard to review this class without seeing its application usage. i think that this PR should include the code that uses this class in the app code.

Reply:
Correct, but i wanted to create small PRs